torch/testing/_internal/common_utils.py

```diff
@@ -38,7 +38,7 @@
 import json
 import __main__  # type: ignore[import]
 import errno
-from typing import cast, Any, Dict, Iterable, Iterator, Optional, Union
+from typing import cast, Any, Dict, List, Iterable, Iterator, Optional, Union
```

Comment:
could put this in alphabetical order but it doesn't really matter
```suggestion
from typing import cast, Any, Dict, Iterable, Iterator, List, Optional, Union
```

Reply:
I ordered them with color: 
![image](https://user-images.githubusercontent.com/31798555/128234474-23b25b23-4867-488c-807e-e888b42de8a6.png)
Is that bad?